src/schema/ecommerce/types/initial_offer_input_type.ts

```diff
@@ -1,19 +1,16 @@
-import {
-  GraphQLInputObjectType,
-  GraphQLNonNull,
-  GraphQLInt,
-  GraphQLID,
-} from "graphql"
+import { GraphQLInputObjectType, GraphQLNonNull, GraphQLID } from "graphql"
+import { MoneyInput } from "schema/fields/money"
+
 export const InitialOfferInputType = new GraphQLInputObjectType({
   name: "InitialOfferOrderInput",
   fields: {
     orderId: {
       type: new GraphQLNonNull(GraphQLID),
       description: "ID of order",
     },
-    amountCents: {
-      type: new GraphQLNonNull(GraphQLInt),
-      description: "Offer amount in cents",
+    offerAmount: {
+      type: MoneyInput,
+      description: "Offer amount in dollars or major unit",
```

Comment:
I think this can just be:

```suggestion
      description: "Offer price",
```

When the developer reads the docs for the `MoneyInput` type they’ll learn about the other details.